dataset_builder/prepare_prompts_json.py

```diff
@@ -56,6 +57,12 @@ def main():
         help="How to translate terminology in prompts: verbatim or reworded"
     )
 
+    args.add_argument(
```

Comment:
We should update the comment for the `--output` argument to say *Target JSONL file*.